openbsw/projectdocumenten/pilotvoorstel.md

```diff
@@ -0,0 +1,87 @@
+# Pilot Voorstel OpenBSW
+De laatste 2 maanden hebben we een Proof of Concept (PoC) gedaan. Deze PoC had een exploratief karakter, hierdoor neemt de PoC nog een aantal aspecten niet mee die wel belangrijk zijn voor het aanbieden van samenwerk diensten. Voorbeelden hiervan zijn toegankelijkheid normen, security & privacy normen, gebruikers input en transparantie richtlijnen. Ook heeft de PoC nog geen uitwerking om te voldoen aan de architectuur van BSW. In de pilot willen we deze aspecten wel meenemen. 
+
+# Voorstel
+Het voorstel is om extra budget beschikbaar te stellen voor pilots onder OpenBSW waarbij we geselecteerde producten uittesten met gebruikersgroepen. Het resultaat aan het einde van de pilot is het volgende
+* Een adviesrapport van het product voor gebruik in Nederlandse samenwerk software suite
+* Een rapport aan welke wet- en regelgeving het product voldoet en welke stappen genomen moeten worden als hij nog niet voldoet.
+* Een samenvattingsrapport die de ervaringen van gebruikers beschrijft
+* Een rapport die de uitwerking beschrijft naar het voldoen aan architectuur van BSW
+* Een rapport die de WCAG oplevert
+
+In de pilot willen we gebruikers toegang gaan geven tot geselecteerde producten uit de Proof of Concept. Hierbij is het belangrijk dat de producten neergezet wordt zodat ze aan de wet- en regelgeving die van toepassing is voldoen. Verder willen we kijken of het product binnen de BSW architectuur past en welke stappen nodig zijn als deze nog niet past.
+
+Met het extra budget creëren we een pilot team. Ook gebruiken we het budget voor het afnemen van computers bij overheidsdatacenters en kunnen we Enterprise licenties (van open source producten) inkopen waar dit nodig is om de doelen van de pilot te behalen.
+
+Ons voorstel voor het pilotproject is om de tijd en middelen vast te zetten, terwijl we de scope flexibel houden. De producten die geprioriteerd worden, kunnen we per product via een nota afstemmen met de stuurgroep, zodat we de nodige flexibiliteit behouden. In deze nota beschrijven we bevindingen van de PoC en beschrijven we ontwikkel, UX-onderzoek, security en operations werkzaamheden.  Zowel het team als de stuurgroep kunnen voorstellen doen voor pilot producten.
+
+Voorbeelden van producten kunnen zijn:
+* Chat (Element synapse)
+* Office (Collabora + NextCloud)
+* Video Bellen (Jitsi/Meet)
+* Notities (Docs)
+* Mail (OpenXChange)
+* AI Interface (OpenWebUI)
+* Projectmanagement (OpenProject)
+* Kennismanagement (XWIki)
+
+# Kaders
+Om het onderzoek waardevol te houden moeten voor alle pilot producten kaders aangehouden worden waarbij Open Strategische Autonomie (OSA) erg belangrijk is voor OpenBSW. Voor de producten gaan we uit van volledige autonomie en volledig open. Hierbij houden we de volgende kaders aan
+
+* Product draait op Rijksdatacenter
+* Containerisatie boven Virtualisatie
+* Opensource tenzij methodiek
```

Comment:
```suggestion
* Open source als voorwaarde
```

Open source tenzij is het kabinetsbeleid. Met Open BSW bewijzen we dat je geen bedrijfseigen software nodig hebt.

Open source is de ondergrens voor technologieën waar we mee werken.